### ISSUE TITLE
zh-cn and zh-tw are not technically ISO639-1

### ISSUE THREAD

**Issue by brianshin22** [Mimino666/langdetect#113]
Any support for changing langdetect output for Chinese to just "zh" to strictly follow ISO639-1 code? In my case, it affects compatibility with python-iso639 package. Thanks!

**Comment by Wolf-Alexander** [comment]
Yes I also think it would be much better to just change it to zh. It also causes problems in my project so I also think, that they should change it
